tests/stress/km/stress_tests_km.cpp

```diff
@@ -1084,6 +1091,296 @@ _print_test_control_info(const test_control_info& test_control_info)
+            bind_program_name += std::to_string(index);
+
+            // Try to get a handle to the 'BindMonitor_Callee<index>' program.
+            bpf_program* callee = bpf_object__find_program_by_name(object, bind_program_name.c_str());
+
+            if (callee == nullptr) {
+                LOG_VERBOSE("({}) - bpf_object__find_program_by_name() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(callee != nullptr);
+            }
+            LOG_VERBOSE("({}) - bpf_object__find_program_by_name() success.", bind_program_name);
+
+            fd_t callee_fd = bpf_program__fd(callee);
+            if (callee_fd < 0) {
+                LOG_VERBOSE("({}) - bpf_program__fd() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(callee_fd > 0);
+            }
+            LOG_VERBOSE("({}) - bpf_program__fd() for callee_fd:{} success.", bind_program_name, callee_fd);
+
+            uint32_t result = bpf_map_update_elem(prog_map_fd, &index, &callee_fd, 0);
+            if (result < 0) {
+                LOG_VERBOSE("({}) - bpf_map_update_elem() failed. errno: {}", bind_program_name, errno);
+                REQUIRE(result == ERROR_SUCCESS);
+            }
+            LOG_VERBOSE("({}) - bpf_map_update_elem() for callee_fd:{} success.", bind_program_name, callee_fd);
+
+        } catch (...) {
+            // No need to terminate. We don't care about user mode issues here.
+        }
+    }
+}
+
+static void
+_invoke_mt_bindmonitor_tail_call_thread_function(thread_context& context)
+{
+    // Test bind.
+    SOCKET socket_handle = INVALID_SOCKET;
+    SOCKADDR_STORAGE remote_endpoint{};
+
+    if (context.role == thread_role_type::MONITOR_IPV4) {
+        remote_endpoint.ss_family = AF_INET;
+    } else {
+        ASSERT(context.role == thread_role_type::MONITOR_IPV6);
+        remote_endpoint.ss_family = AF_INET6;
+    }
+
+    uint16_t remote_port = SOCKET_TEST_PORT + static_cast<uint16_t>(context.thread_index);
+    using sc = std::chrono::steady_clock;
+    auto endtime = sc::now() + std::chrono::minutes(context.duration_minutes);
+    while (sc::now() < endtime) {
+
+        // Now send out a small burst of TCP 'bind' attempts for the duration of the test.  We do this to increase
+        // the probability of a collision between the program invocation and the extension being restarted at the same
+        // time.
+        constexpr uint32_t BURST_SIZE = 4;
+
+        for (uint32_t i = 0; i < BURST_SIZE; i++) {
+            // Create a socket.
+            socket_handle = WSASocket(remote_endpoint.ss_family, SOCK_STREAM, IPPROTO_TCP, nullptr, 0, 0);
+            REQUIRE(socket_handle != INVALID_SOCKET);
+
+            INETADDR_SETANY(reinterpret_cast<PSOCKADDR>(&remote_endpoint));
+            SS_PORT(&remote_endpoint) = htons(remote_port);
+
+            // Forcefully bind to the same port in use using socket option SO_REUSEADDR.
+            // Use-case: multicast sockets bind to same port.
```

Comment:
`SO_REUSEADDR` is used for unicast sockets as well (https://lwn.net/Articles/542629/).  I'd recommend removing the 'use-case:...' line.